kernel/src/net/socket/ip/stream/util.rs

```diff
@@ -35,6 +44,64 @@ impl Default for TcpOptionSet {
     }
 }
 
+const HZ: u32 = 100;
```

Comment:
`HZ` should not appear here. In fact, if you always write something like `TCP_TIMEOUT_INIT / HZ`, you should be able to remove the `HZ` constant.

Meanwhile, `TCP_TIMEOUT_INIT` might be better to be a `Duration` (instead of a `u32`).